docs/mattermost-operator/mattermost-operator.yaml

```diff
@@ -5650,7 +5696,7 @@ spec:
           value: "20"
         - name: REQUEUE_ON_LIMIT_DELAY
           value: 20s
-        image: mattermost/mattermost-operator:v1.16.0
+        image: mattermost/mattermost-operator:latest
```

Comment:
should we point here v1.17?

Reply:
This is autogenerated by kustomize, bump to 1.17 will be done with the next direct commit together with some other version changes.